tests/components/anova/test_sensor.py

```diff
@@ -28,34 +22,25 @@ async def test_sensors(hass: HomeAssistant, anova_api: AnovaApi) -> None:
     assert hass.states.get("sensor.anova_precision_cooker_cook_time").state == "0"
     assert (
         hass.states.get("sensor.anova_precision_cooker_heater_temperature").state
-        == "20.87"
+        == "22.37"
```

Comment:
Why are the values changing?

Reply:
Values are changing because the test data has changed slightly since it is using a different protocol.